src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    {person.age > 0
+      && <p className="Person__age">{`I am ${person.age}`}</p>
+    }
+
+    {person.isMarried === true
+      ? <p className="Person__partner">{`${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`}</p>
+      : <p className="Person__partner">I am not married</p>
```

Comment:
let`s move something into a variable to avoid nested ternary operator

Reply:
ok